src/typings/react-feature-toggles/index.d.ts

```diff
@@ -1,78 +1,12 @@
 declare module '@paralleldrive/react-feature-toggles' {
```

Comment:
We should be able to delete this file completely correct? there shouldn't be any need for a typings file for this package anymore.

Reply:
Yes. I added it back trying to see if we could override the typings. I meant to delete it again.